concepts/protocol-assets/newton/index.md

```diff
@@ -18,3 +20,22 @@ Newton is bonded by the owner staking the token to a [validator](/glossary/#vali
+
+Inflation emissions to stakers and validators are crucial for incentivizing network security, ensuring that those contributing to the Newton [participation rate](/glossary/#participation-rate) are adequately rewarded for securing the blockchain. Newton inflation subsidizes the cost and risk of staking during the early stages of the network when transactional activity is low and Auton [staking rewards](/glossary/#staking-rewards) earned from transaction fees alone may not incentivize a sufficiently high staking participation rate. Inflation rewards also help decentralize the network by encouraging a broader base of Newton [stakeholders](/glossary/#stakeholder) to stake their tokens, reducing the risk of centralization. Additionally, these rewards can provide a predictable economic model, attracting long-term investment and commitment from the community.
+
+Autonity’s inflation mechanism is a carefully crafted deterministic emission schedule that is determined based on the number of circulating tokens, irrespective of how many tokens are staked or not. This means that inflation yield for stake [delegations](/glossary/#delegation) is directly impacted by the [participation rate](/glossary/#participation-rate) (i.e. the ratio of staked to non-staked Newton), encouraging staking until a market determined acceptable yield is achieved.
+
+A total of 40 million NTNs will be set aside for inflation rewards from a total supply of 100 million NTNs, ensuring a significant amount of value accrual to stakers over the lifetime of Autonity. NTN inflation rewards are minted and distributed to stakers at the end of each epoch.

The exact parameters of the Mainnet inflation mechanism will be determined through active engagement with the community, but the overall curve will follow a slow decrease in inflation from genesis until ${\sim} Month 36$, after which the inflation emissions will decay asymptotically to zero far into the future. This can best be summarised by an example curve, as seen below.
+
+![Figure 1, example inflation curve. Note that parameters are purely illustrative and the Mainnet curve will likely change.](./images/example-inflation-curve.png)
+
+Please note that in the graph above, the y-axis percentage returns assume a 100% [participation rate](/glossary/#participation-rate) (i.e. all tokens are staked). To model for yourself the expected inflation yield, divide the
value on the y-axis by an estimated participation rate (e.g. 0.7 for an expected participation rate of 70%, resulting in an inflation yield of 14.3% at genesis for an inflation rate of 10%).
+
+For the [Piccadilly Circus Games Competition ](https://game.autonity.org/) on the [Piccadilly Testnet](/networks/testnet-piccadilly/), a slightly different inflation schedule has been used to aid in testing. The Round 6 emission schedule can be seen below:
+
+![Figure 2, inflation curve for Round 6 of the Piccadilly Circus Games Competition.](./images/r6-inflation.png)
+
```

Comment:
> a slightly different inflation schedule has been used to aid in testing. The Round 6 emission schedule can be seen below

Actually, the two graphs are the same (the inflation rate used for R6). We can keep the first one with the percentages. But can shortly say that the R6 curve is designed for testing and should differ from the final curve for MainNet.